horovod/mxnet/mpi_ops.py

```diff
@@ -127,6 +135,8 @@ def allgather(tensor, name=None):
     Arguments:
         tensor: A tensor to allgather.
         name: A name of the allgather operation.
+        priority: The priority of this operation. Higher priority operations are
+                  likely to be executed before other actions.
```

Comment:
nit: change the word "actions" to "operations" to be consistent

Reply:
Done